### ISSUE TITLE
requirements.txt

### ISSUE THREAD

**Issue by Decade-rider** [jacobortiz/adaptive-bc#1]
This packages list seems don't be installed directly. This is my input   conda create --name adaptive_bc --file requirements.txt 
output CondaError: Error reading file, file should be a text file containing packages
And I have create a conda env ,named as adaptive_bc, activate this env ,then pip install -r requirements.txt 
output ERROR: Invalid requirement: 'blas=1.0=mkl' (from line 4 of requirements.txt)
Hint: = is not a valid operator. Did you mean == ?

**Comment by Decade-rider** [comment]
Thanks your awesome project very much，I have successfuly run the DW model when I created a test.py。I am a first-year graduate student from China. My research direction is the point of view dynamic model and I seek to use it for empirical evidence.And another question is that if I want to use this model for real study,such as study how do user‘s comments change in Twitter.could you give me some advice?Is there a good form for using this model to study real world?

**Comment by jacobortiz** [comment]
Yes, I have encountered this issue as well, I'm not sure what is causing it but will get back to you.

**Comment by Decade-rider** [comment]
好的，这个貌似影响不大，我已经试运行成功了一个test.py，在python3.9环境下。此外，之前也给您的geogle邮箱发了封邮件，询问如何将该项目用于实证研究的相关问题。希望您能给予一点帮助，非常感谢。



---Original---
From: ***@***.***&gt;
Date: Wed, Mar 27, 2024 01:17 AM
To: ***@***.***&gt;;
Cc: "kamen ***@***.******@***.***&gt;;
Subject: Re: [jacobortiz/adaptive-bc] requirements.txt (Issue #1)




 
Yes, I have encountered this issue as well, im not sure what is causing it but will get back to you.

**Comment by jacobortiz** [comment]
yes this branch is on python 3.8.5, currently updating it to latest version of python.
